db/models.py

```diff
@@ -1 +1,37 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True, null=True)
```

Comment:
It would be better to always define the `__str__` method when creating a new model.